### ISSUE TITLE
Pushing to Release

### ISSUE THREAD

**Issue by Sixdsn** [Sixdsn/terra-terminal#89]
I think we should focus on major bugs and reliability to be able to merge develop and master in order to deliver a new release of terra ASAP (depending of our free time of course ;) )

We have to use Milestones to track important patches that should be in the release
- Milestone Testing1.9 is used for the actual version in the develop branch
- Milestone Relase2.0 will be the merge of Testing1.9 + the debug/tests
- Milestone Future is used for non mandatory patches
  We have to create issues for each patch (except code cleaning) to track the progression.

@SchnWalter what do you think about that?

**Comment by SchnWalter** [comment]
For internationalization (#63) we need to do a lot more than just fix the parser;  we also have a lot of strings that don't go through the translations interface.

Do we want this i18n in the next release. I believe many strings will change if we do more refactoring, maybe we should be more vigilant when we write/change code and always make sure that i18n work properly, and this way, in a future release we won't have so much work.

**Comment by Sixdsn** [comment]
I think we should completly interger i18n for the next release and then take care on how we use it in the next refactors

**Comment by SchnWalter** [comment]
I don't think that we should release under the same name, we should try and lookup this guys email and maybe he will answer and create an organization here, and make us admins so that we can carry on with the project... otherwise we might need to rename it.

Oh, and I've seen you are working on the Github wiki, I would suggest dropping it in favour of simple markdown files in a `docs` folder, and this way we can later use those files to build a `man` page or something... And if Github is no longer an option, we won't need to worry about migrating the wiki pages.

**Comment by Sixdsn** [comment]
Well as this is a fork of the original **terra** i don't want to loose the name, we might just need to do more communication about this version, specially when we will have closed every major issues (there are only a few left).
We can export the wiki markdown files so right now i prefer to keep the Wiki up-to-date and if we need another platform later we will be able to move everythin easily.

**Comment by SchnWalter** [comment]
Maybe we should track down his email, and try to convince him to pass the project, if he is answering.

**Comment by Sixdsn** [comment]
I sould mailed him with all informations and what we want to do, i wish we will get an answer soon
@ozcanesen
